src/github/dto/create.org.dto.ts

```diff
@@ -0,0 +1,82 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsString,
+  IsEmail,
+  IsEnum,
+  IsArray,
+  IsBoolean,
+  IsOptional,
+  Length,
+} from 'class-validator';
+import { OrganizationStatus } from 'src/common/constants';
+
+export class CreateOrgDto {
+  @ApiProperty({
+    description: 'The host of the organization (e.g., GitHub)',
+    example: 'github',
+    default: 'github',
+  })
+  @IsString()
+  @Length(1, 255)
+  host: string;
+
+  @ApiProperty({
+    description: 'The unique name of the organization',
+    example: 'example-org',
+  })
+  @IsString()
+  @Length(1, 255)
+  organizationName: string;
+
+  @ApiProperty({
+    description: 'The unique username of the admin',
+    example: 'admin-user',
+  })
+  @IsString()
+  @Length(1, 255)
+  adminUserName: string;
+
+  @ApiProperty({
+    description: 'The unique email of the admin',
+    example: 'admin@example.com',
+  })
+  @IsString()
+  @Length(1, 255)
+  adminEmail: string;
+
+  @ApiProperty({
+    description: 'A list of active users',
+    example: ['user1', 'user2'],
+    type: [String],
+  })
+  @IsArray()
+  @IsString({ each: true })
+  @IsOptional()
+  activeUsers: string[];
+
+  @ApiProperty({
+    description: 'A list of inactive users',
+    example: ['user3', 'user4'],
+    type: [String],
+  })
+  @IsArray()
+  @IsString({ each: true })
+  @IsOptional()
+  inActiveUsers: string[];
```

Comment:
PullProofAI

The type definition for `activeUsers` and `inActiveUsers` should be corrected to specify an array of strings.

```diff
-    type: [String],
+    type: [String]
```

#### Issue: Typo in the comment for `allowAllUsers`


<!-- This is an auto-generated comment by PullProofAI -->